src/crimson/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ set_target_properties(crimson::cflags PROPERTIES
   INTERFACE_LINK_LIBRARIES Seastar::seastar)
 
 set(crimson_common_srcs
+  admin/admin_socket.cc
```

Comment:
could you clean up the commit message a little bit? to be specific, would be better if the lines between "A note regarding code format:" and "Signed-off-by: Ronen Friedman <rfriedma@redhat.com>" are removed.

Reply:
done